apps/deploy-web/playwright.config.ts

```diff
@@ -36,17 +33,17 @@ export default defineConfig({
     {
       name: "chromium",
       use: { ...devices["Desktop Chrome"] }
-    },
+    }
 
-    {
-      name: "firefox",
-      use: { ...devices["Desktop Firefox"] }
-    },
+    // {
+    //   name: "firefox",
+    //   use: { ...devices["Desktop Firefox"] }
+    // },
 
-    {
-      name: "webkit",
-      use: { ...devices["Desktop Safari"] }
-    }
+    // {
+    //   name: "webkit",
+    //   use: { ...devices["Desktop Safari"] }
+    // }
```

Comment:
✂️

Reply:
Yea, I'll remove it because chrome extensions don't work with firefox or safari.